db/schema.rb

```diff
@@ -13,7 +13,6 @@
 ActiveRecord::Schema.define(version: 2021_09_14_144638) do
 
   # These are extensions that must be enabled in order to support this database
-  enable_extension "pg_stat_statements"
```

Comment:
This line seemed to have been introduced in a commit with unrelated changes. We don't seem to be using it and it was causing problems, so I've removed it.